docs/faq/items/python2_support.md

```diff
@@ -0,0 +1,6 @@
+# Does Black support Python 2?
+
+For formatting, yes! [Install](../../getting_started.md#installation) with the `python2`
```

Comment:
I *love* the "For formatting, yes!" phrase, I need to use more exclamation marks in my writing.